__mocks__/@zextras/carbonio-shell-ui.ts

```diff
@@ -92,19 +84,23 @@ export const useBoard: <T>() => Board<T> = () => {
 	};
 };
 
+export const getBoardById: typeof shell.getBoardById = jest.fn();
```

Comment:
Avoid jest.fn here in the module mocks, it is not needed and might cause problems due to the restore of the mocks. Just assign an empty `() => undefined` function